.php_cs.dist

```diff
@@ -6,11 +6,43 @@ $finder = PhpCsFixer\Finder::create()
 ;
 
 return PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
     ->setRules([
         '@Symfony' => true,
+        '@Symfony:risky' => true,
+        'array_syntax' => [
+            'syntax' => 'short',
+        ],
+        'braces' => [
+            'allow_single_line_closure' => true,
+        ],
+        'declare_strict_types' => true,
+        'modernize_types_casting' => true,
+        // 'native_function_invocation' => true,
+        'no_extra_consecutive_blank_lines' => [
+            'break',
+            'continue',
+            'curly_brace_block',
+            'extra',
+            'parenthesis_brace_block',
+            'return',
+            'square_brace_block',
+            'throw',
+            'use',
+        ],
+        'no_unreachable_default_argument_value' => true,
+        'no_useless_else' => true,
+        'no_useless_return' => true,
         'ordered_imports' => true,
+        // 'phpdoc_add_missing_param_annotation' => [
+        //     'only_untyped' => false,
+        // ],
```

Comment:
@dunglas I want to get this in too... But this is gonna require quite a bit of manual work. :laughing: